invenio_search/api.py

```diff
@@ -65,8 +66,13 @@ def search(self, user_info=None, collection=None, **kwargs):
 
         for walker in search_walkers():
             query = query.accept(walker)
-        return Results(query)
 
+        index = cfg["SEARCH_ELASTIC_COLLECTION_INDEX_MAPPING"].get(
```

Comment:
I agree about "index" being different from "collection",  see my comments in https://github.com/inveniosoftware/invenio/pull/3450#issuecomment-132503979.

Is it a need for INSPIRE to look in HEP and in JOBS at the same time?  They used to be different "databases", available on different "tabs", that are not usually searched together...